SDCAlertView.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,7 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		0BA53BDB1F31D23D003854F3 /* UIView+Accessibility.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0BA53BDA1F31D23D003854F3 /* UIView+Accessibility.swift */; };
```

Comment:
Please make sure these files are sorted alphabetically. This could just be Xcode messing that up, but looks like it might not be.